docs/getting-started/install-configure.md

```diff
@@ -248,7 +248,7 @@ kubectl crossplane install configuration registry.upbound.io/xp/getting-started-
 
 Wait until all packages become healthy:
 ```
-watch kubectl get pkg
+kubectl get pkg --watch
```

Comment:
Does this work? I thought you couldn't `--watch` `pkg` because it's a category.

FWIW the original `watch kubectl` variant does work - it just depends on the distinct `watch` command being installed.